applications/nrf_desktop/src/modules/ble_discovery.c

```diff
@@ -101,8 +115,8 @@ static void start_hids_discovery(struct bt_conn *conn)
 {
```

Comment:
This function is almost identical as start_discovery. All difference as related with state of module (it means what GATT characteristic is going to be discovered)

Reply:
Other callback is used here. I also wanted to separate this one from reading other parameters.

Reply:
both functions has the same goal - start GATT discovery. Why make two functions if one is possible? Can't it provide callback conditionally?

Reply:
Discussed offline. This will cause problems with `static const struct` initialization.